code/modules/power/supermatter/supermatter.dm

```diff
@@ -188,6 +188,8 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 
 /obj/machinery/power/supermatter_crystal/examine(mob/user)
 	. = ..()
+	if(isobserver(user)) // ghosts don't have mind
+		return .
```

Comment:
If the runtime is thrown by mobs that don't have a mind, then check for the user not having a mind and not if they are an observer or not